src/util/formatTechRecord.ts

```diff
@@ -48,11 +51,14 @@ export const formatTechRecord = (techRecordWithoutArrays: object) => {
     if (!/_\d+/.test(key)) {
       // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, security/detect-object-injection, @typescript-eslint/no-unsafe-member-access
       formattedTechRecord[key] = value;
+    } else if (/techRecord_\w+_\w+_\d+/.test(key)) {
```

Comment:
Wonder if there is a cool way to do this without if/else now, but not sure the time needed is worth it